messaging/src/main/scala/uk/ac/wellcome/messaging/worker/result/Result.scala

```diff
@@ -0,0 +1,23 @@
+package uk.ac.wellcome.messaging.worker.result
+
+sealed trait Result[Summary] {
+  val id: String
+  val summary: Summary
+
+  def pretty(resultType: String) =
+    s"""
+       |$resultType: $id
+       |$summary
+       """.stripMargin
```

Comment:
👍 for including an ID in this string

If we're thinking about doing more stuff with logging, maybe a pretty-printed string isn’t ideal? Would a small JSON blob be easier for automated parsing? Not a strong feeling, just an idea.

Reply:
We should definitely think about how to record these summaries.